From 9ea8e0284a1904e7b02dfa70bd0f7ef0a05daf3e Mon Sep 17 00:00:00 2001
From: tgeisler <tgeisler@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 11 Sep 2013 14:34:29 +0000
Subject: [PATCH 0474/1077] MANTIS	0029017: Platform Setup: Remove
 generated files and directories in the user local folder with an
 uninstallation change mcontrol file location

git-svn-id: svn://localhost/SelfServiceCommon/trunk@580 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/Controller/inc/mcontrol_def.h           | 10 +++++++---
 SelfServiceCommon/Massai/cpp/Controller/src/server.cpp |  5 +++++
 2 files changed, 12 insertions(+), 3 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h b/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
index 951fd452..b5ca8d9c 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
+++ b/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
@@ -9,15 +9,19 @@
  *
  ************************************************************************/
 
-#define FILE_VER_BIN     1,0,2,11
-#define FILE_VER_STR     "FileVersion", "1.0.2.11\0"
+#define FILE_VER_BIN     1,0,2,12
+#define FILE_VER_STR     "FileVersion", "1.0.2.12\0"
 
 /*! \page mControl.exe - platform startup, starts mudpate.exe, mstarter.exe; servant for systemControl interface
 
-@version 1.0.2.11
+@version 1.0.2.12
 
 \section change1 Change history
 
+\date 11.09.2013
+\li V1.0.2.12
+\li modification - Mantis  0029017: Platform Setup: Remove generated files and directories in the user local folder with an uninstallation - change mcontrol file location 
+
 \date 25.07.2013
 \li V1.0.2.11
 \li bugfix - SteT: Mantis  0028216: mcontrol: do not create logging client before logging servant is started
diff --git a/SelfServiceCommon/Massai/cpp/Controller/src/server.cpp b/SelfServiceCommon/Massai/cpp/Controller/src/server.cpp
index bc82c7b3..4ac8b669 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/src/server.cpp
+++ b/SelfServiceCommon/Massai/cpp/Controller/src/server.cpp
@@ -682,6 +682,11 @@ std::string controllerServant::getRebootFileName()
         {
           PathAppend(app_path,"materna\\");
         }
+        buf.assign(app_path); buf.append("MIPS");
+        if(PathIsDirectory(buf.c_str()))
+        {
+          PathAppend(app_path,"MIPS\\");
+        }
         userDir.assign(app_path).append(REBOOT_FILE_NAME);
       } else
       userDir.assign(".\\"REBOOT_FILE_NAME);
-- 
2.41.0.windows.1

